README.md

```diff
@@ -17,101 +17,82 @@
+There are no releases of **CUBOS.** so far. We will update this section once we release official binaries.
+<!--Official binaries for **CUBOS.** can be found
+on the [releases](https://github.com/GameDevTecnico/cubos/releases) page.-->
 
-##### Visual Studio Code
+### Building from the source code
 
-1. Install *Microsoft's* *[CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools)* extension for *Visual Studio Code*.
-2. Open the repository folder in *Visual Studio Code*. It will automatically scan your project.
-3. On the ```Would you like to configure project 'cubos'?``` prompt thrown by *CMake Tools*, press **Yes**
+#### Dependencies
 
-<p align="center">
-    <img src="docs/images/CMakeToolsPrompt.png" alt="```Would you like to configure project 'cubos'?``` prompt">
-</p>
+The following dependencies are used to compile **CUBOS.**:
 
-4. Then pick a kit for **CUBOS.** This is your preferred compiler.
+| Name                                               | Importance         | Submodule Path        | Installed Separately |
+| -------------------------------------------------- | ------------------ | --------------------- | -------------------- |
+| [CMake](https://cmake.org/)                        | Essential          | -                     | Yes                  |
+| [glad](https://github.com/Dav1dde/glad)            | Essential          | -                     | No                   |
+| [glfw](https://github.com/glfw/glfw)               | Essential          | `core/lib/glfw`       | Optionally           |
+| [glm](https://github.com/g-truc/glm)               | Essential          | `core/lib/glm`        | Optionally           |
+| [fmt](https://github.com/fmtlib/fmt)               | Essential          | `core/lib/fmt`        | Optionally           |
+| [spdlog](https://github.com/gabime/spdlog)         | Essential          | `core/lib/spdlog`     | Optionally           |
+| [yaml-cpp](https://github.com/jbeder/yaml-cpp)     | Essential          | `core/lib/yaml-cpp`   | Optionally           |
+| [googletest](https://github.com/google/googletest) | Required for tests | `core/lib/googletest` | Optionally           |
 
-<p align="center">
-    <img src="docs/images/CMakeKitPrompt.png" alt="Kit prompt">
-</p>
+Dependencies marked as *Essential* are required to compile the engine.
+**CUBOS.** uses [CMake](https://cmake.org/) as its build system, so you must install it to compile the engine.
 
-5. Finally, on the bar at the bottom, press **Build**. The first time it compiles will take longer.
+Most dependencies come as submodules, so, you may choose whether to install them separately or not. If you choose to install some or all of them separately, you can do so easily by using the package manager of your choice. If you use Windows (ew), it should be easier to simply install all of them with the engine by cloning
```

Comment:
Remove discrimination targetted towards all binarily impaired users, **CUBOS.** should strive to be an inclusive project for people of all diferent ways of life.